GL_HALF_FLOAT is not support on windows while replaying

GAPID Version: 1.3.1:44c6681a66df5e84e2cd8e56fbd8f968be2fafd6
OS: windows 7 amd64


GL_HALF_FLOAT is GLES3.0 supported 
but when i select  windows  PC (Windows 7 Intel HD Graphic 530 )  as replay device, it show error log as follow 

```
panic: Unknown unsized format: GL_RGBA, GL_HALF_FLOAT [recovered]
	panic: Panic at command 7537:glTexImage2D(target: GL_TEXTURE_2D, level: 0, internalformat: 6408, width: 2, height: 2, border: 0, format: GL_RGBA, type: GL_HALF_FLOAT, data: 0x00000074f8f0d500):
Unknown unsized format: GL_RGBA, GL_HALF_FLOAT [recovered]
	panic: Panic at command 7537:glTexImage2D(target: GL_TEXTURE_2D, level: 0, internalformat: 6408, width: 2, height: 2, border: 0, format: GL_RGBA, type: GL_HALF_FLOAT, data: 0x00000074f8f0d500):
Unknown unsized format: GL_RGBA, GL_HALF_FLOAT

goroutine 25 [running]:
github.com/google/gapid/core/app/crash.Crash(0x512ed00, 0xc02814b3f0)
	core/app/crash/crash.go:89 +0xae
github.com/google/gapid/core/app/crash.handler()
	core/app/crash/crash.go:56 +0x59
panic(0x512ed00, 0xc02814b3f0)
	GOROOT/src/runtime/panic.go:513 +0x1c7
github.com/google/gapid/gapis/api.ForeachCmd.func1(0xc0476334c0, 0xc0476334f0)
	gapis/api/cmd_foreach.go:43 +0x159
panic(0x512ed00, 0xc02814b3e0)
	GOROOT/src/runtime/panic.go:513 +0x1c7
github.com/google/gapid/gapis/api/gles.getSizedFormatFromTuple(0x140b00001908, 0x0)
	gapis/api/gles/image.go:45 +0x1cb
github.com/google/gapid/gapis/api/gles.(*textureCompat).convertFormat(0xc051caac30, 0x62ea020, 0xc0520be9c0, 0xde1, 0xc0476323d0, 0xc0476323a0, 0xc047632370, 0x62306e0, 0xc05221fce0, 0x1d71, ...)
	gapis/api/gles/texture_compat.go:138 +0x718
github.com/google/gapid/gapis/api/gles.compat.func3(0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x62306e0, 0xc05221fce0)
	gapis/api/gles/compat.go:660 +0x15b9
github.com/google/gapid/gapis/api/transform.transform.Transform(0x5db7779, 0x6, 0xc052020210, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x62306e0, 0xc05221fce0)
	gapis/api/transform/transforms.go:75 +0x7c
github.com/google/gapid/gapis/api/transform.TransformWriter.MutateAndWrite(0xc01b8721c0, 0x6230720, 0xc0520f47e0, 0x62306e0, 0xc05221fce0, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0)
	gapis/api/transform/transforms.go:98 +0x8a
github.com/google/gapid/gapis/api/transform.(*Tasks).Transform(0xc05221f9b0, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x62306e0, 0xc05221fd10)
	gapis/api/transform/tasks.go:46 +0x15a
github.com/google/gapid/gapis/api/transform.TransformWriter.MutateAndWrite(0xc01b8721c0, 0x6225060, 0xc05221f9b0, 0x62306e0, 0xc05221fd10, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0)
	gapis/api/transform/transforms.go:98 +0x8a
github.com/google/gapid/gapis/api/gles.undefinedFramebuffer.func1(0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x62306e0, 0xc05221fd40)
	gapis/api/gles/undefined_framebuffer.go:31 +0xa6
github.com/google/gapid/gapis/api/transform.transform.Transform(0x5dcad9f, 0x10, 0xc0520f4680, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x62306e0, 0xc05221fd40)
	gapis/api/transform/transforms.go:75 +0x7c
github.com/google/gapid/gapis/api/transform.TransformWriter.MutateAndWrite(0xc01b8721c0, 0x6230720, 0xc0520f46a0, 0x62306e0, 0xc05221fd40, 0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0)
	gapis/api/transform/transforms.go:98 +0x8a
github.com/google/gapid/gapis/resolve/dependencygraph.(*DeadCodeElimination).Flush.func1(0x62ea020, 0xc0520be9c0, 0x1d71, 0x6321a20, 0xc000a1cde0, 0x0, 0x0)
	gapis/resolve/dependencygraph/dead_code_elimination.go:88 +0x86
github.com/google/gapid/gapis/api.ForeachCmd(0x62ea020, 0xc0520be4e0, 0xc01040e000, 0xa228, 0xbc00, 0xc047633598, 0x0, 0x0)
	gapis/api/cmd_foreach.go:54 +0x277
github.com/google/gapid/gapis/resolve/dependencygraph.(*DeadCodeElimination).Flush(0xc0520f4660, 0x62ea020, 0xc0520be450, 0x62306e0, 0xc05221fd70)
	gapis/resolve/dependencygraph/dead_code_elimination.go:87 +0x2f1
github.com/google/gapid/gapis/api/transform.Transforms.Transform(0xc024ad7c80, 0x5, 0x8, 0x62ea020, 0xc05221f8f0, 0xc0476337c0, 0x0, 0x0, 0x622a2a0, 0xc04c90f600)
	gapis/api/transform/transforms.go:44 +0x314
github.com/google/gapid/gapis/api/gles.API.Replay(0x62ea020, 0xc05221f860, 0xc05221f0e0, 0xc05221eea0, 0x57caaa0, 0xc037def940, 0x0, 0x0, 0xc0520f4360, 0x1, ...)
	gapis/api/gles/replay.go:275 +0x5e9
github.com/google/gapid/gapis/replay.(*manager).execute.func1()
	gapis/replay/batch.go:330 +0x255
github.com/google/gapid/core/app/benchmark.(*DurationCounter).Time(0xc00021eb10, 0xc0476340b0)
	core/app/benchmark/counter.go:163 +0x84
github.com/google/gapid/gapis/replay.(*manager).execute(0xc0002122c0, 0x62ea020, 0xc05221f260, 0x63313c0, 0xc0000df550, 0x1140f5220b5ef5d7, 0x675dbb66221de12d, 0x107bb14730ed0464, 0xb928c0d6a359bff8, 0x47aa6486b84ae308, ...)
	gapis/replay/batch.go:326 +0xd6e
github.com/google/gapid/gapis/replay.(*manager).batch.func1(0x63313c0, 0xc0000df550, 0xc047634460, 0xa359bff8107bb147, 0xb84ae308b928c0d6, 0xb5ef5d747aa6486, 0x221de12d1140f522, 0x30ed0464675dbb66, 0x57caaa0, 0xc037def940, ...)
	gapis/replay/batch.go:86 +0x425
github.com/google/gapid/gapis/replay.(*manager).batch(0xc0002122c0, 0x62ea020, 0xc05221ee70, 0xc0520f4320, 0x1, 0x1, 0x0, 0x0, 0x57e12a0, 0xc01f7cf4a0, ...)
	gapis/replay/batch.go:87 +0x447
github.com/google/gapid/gapis/replay.(*manager).batch-fm(0x62ea020, 0xc0000ff8c0, 0xc0520f4320, 0x1, 0x1, 0x0, 0x0, 0x57e12a0, 0xc01f7cf4a0, 0x3)
	gapis/replay/manager.go:146 +0x94
github.com/google/gapid/gapis/replay/scheduler.(*bin).exec(0xc024ad7a00, 0x62ea020, 0xc0000ff8c0, 0xc0000df580)
	gapis/replay/scheduler/scheduler.go:247 +0x278
github.com/google/gapid/gapis/replay/scheduler.(*Scheduler).run(0xc000212560, 0x62ea020, 0xc0000ff8c0)
	gapis/replay/scheduler/scheduler.go:180 +0x440
github.com/google/gapid/gapis/replay/scheduler.New.func1()
	gapis/replay/scheduler/scheduler.go:74 +0x43
github.com/google/gapid/core/app/crash.Go.func1(0xc000212580)
	core/app/crash/crash.go:65 +0x4a
created by github.com/google/gapid/core/app/crash.Go
	core/app/crash/crash.go:63 +0x46
```

--- a/gapis/api/gles/api/image_format.api
+++ b/gapis/api/gles/api/image_format.api
@@ -419,7 +419,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_RED_EXT: switch type {
       case GL_UNSIGNED_BYTE: GL_R8
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_R16F
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_R16F
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_R32F
       default: GL_NONE
@@ -428,7 +428,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_RG_EXT: switch type {
       case GL_UNSIGNED_BYTE: GL_RG8
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_RG16F
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_RG16F
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_RG32F
       default: GL_NONE
@@ -439,7 +439,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
       @if(Extension.GL_EXT_texture_type_2_10_10_10_REV)
       case GL_UNSIGNED_INT_2_10_10_10_REV: GL_RGB10
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_RGB16F
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_RGB16F
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_RGB32F
       default: GL_NONE
@@ -451,7 +451,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
       @if(Extension.GL_EXT_texture_type_2_10_10_10_REV)
       case GL_UNSIGNED_INT_2_10_10_10_REV: GL_RGB10_A2
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_RGBA16F
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_RGBA16F
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_RGBA32F
       default: GL_NONE
@@ -470,7 +470,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_LUMINANCE_ALPHA: switch type {
       case GL_UNSIGNED_BYTE: GL_LUMINANCE8_ALPHA8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_LUMINANCE_ALPHA16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_LUMINANCE_ALPHA16F_EXT
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_LUMINANCE_ALPHA32F_EXT
       default: GL_NONE
@@ -478,7 +478,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_LUMINANCE: switch type {
       case GL_UNSIGNED_BYTE: GL_LUMINANCE8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_LUMINANCE16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_LUMINANCE16F_EXT
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_LUMINANCE32F_EXT
       default: GL_NONE
@@ -486,7 +486,7 @@ sub GLenum GetSizedFormatFromTuple(GLenum unsizedFormat, GLenum type) {
     case GL_ALPHA: switch type {
       case GL_UNSIGNED_BYTE: GL_ALPHA8_EXT
       @if(Extension.GL_OES_texture_half_float)
-      case GL_HALF_FLOAT_OES: GL_ALPHA16F_EXT
+      case GL_HALF_FLOAT, GL_HALF_FLOAT_OES: GL_ALPHA16F_EXT
       @if(Extension.GL_OES_texture_float)
       case GL_FLOAT: GL_ALPHA32F_EXT
       default: GL_NONE

